### ISSUE TITLE
Scaffold editor fallback

### ISSUE THREAD

**Issue by lc-thomasberger** [adaptlearning/adapt_authoring#2221]
## Affected Area
Backbone Forms override

## Requested Feature
Plugin authors should be able to define a fallback editor.
Instead of breaking if no supported input editor is found, scaffold should fallback to Text editor or a generic editor (in the future) that is well supported.

## Current Workaround
None

## Additional Information
https://github.com/adaptlearning/adapt_authoring/pull/2204 <-- user editor
https://github.com/adaptlearning/adapt_authoring/pull/2156 <-- assetItems editor

**Comment by taylortom** [comment]
:+1: I think even displaying some kind of `could not be displayed, please contact your administrator` message would in place of the missing editor would be fine.

**Comment by lc-thomasberger** [comment]
👍 , will add highlighting to the PR.
Being able to see and edit the value would be good though for power users.
We can experiment with this and decide what route to go.

**Comment by taylortom** [comment]
Yeah true, should probably be a code editor by default then, as that's the only one that would allow you to set a correctly formatted output.
